src/cpp/benders/output/JsonWriter.cpp

```diff
@@ -1,5 +1,7 @@
 #include "JsonWriter.h"
 #include "config.h"
+#include <iostream>
+#include <fstream>
```

Comment:
Are these really useful ?

Reply:
**output** lib is intended to be only standard library dependant.
yes for fstream (use of std::ofstream)-> we write on out.json file.
iostream to provide std::cout.